routes/api.php

```diff
@@ -20,3 +21,4 @@
 });
 
 Route::get('token/{idToken}', [Token::class, 'handle']);
+Route::get('token/{idToken}/integration/{id}', [Access::class, 'handle']);
```

Comment:
Shouldn't we postfix these controller names with Controller like we did in `web.php`?

e.g. `AccessController` and `TokenController`

Reply:
Those names are indeed not consistent. I will add the `Controller` postfix and also change the already existing ones like `Login, `Logout` and `Callback`

The names `Login`, `Logout`, and `Callback` did follow the Auth0 SDK conventions.